packages/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts

```
@@ -31,9 +31,9 @@ group = "expo.modules"
 
 gradlePlugin {
   plugins {
-    register("expoModulesLibrary") {
-      id = "expo-modules-library"
-      implementationClass = "expo.modules.plugin.ExpoModulesLibraryPlugin"
+    register("expoModulesGradlePlugin") {
+      id = "expo-module-gradle-plugin"
```

```suggestion
      id = "expo-modules-gradle-plugin"
```
Not sure if this is intended?

---

It was intentional. Since it's a plugin that you apply to a specific module, naming it `expo-modules-module-gradle-plugin` seems very odd.